lms/djangoapps/courseware/masquerade.py

```diff
@@ -69,6 +69,46 @@ def __setstate__(self, state):
         """
         self.__init__(**state)
 
+    def get_status(self, available):
+        """
+        Retrieve a relevant masquerade status message,
+        ala "You are masquerading as.."
+
+        Returns: a relevant status message, if one exists,
+            else, return None
+
+        Note: We intentionally omit a message for 'staff' users:
+            this is the default status, so we don't need to advertise it.
```

Comment:
If we were to decide we _do_ want this behavior (I think it's too verbose; I don't know that "masquerading as staff" is functionally different than just being staff...), just add a case below in the `if`/`elif`.